### DISCUSSION TITLE
When removing an item from trash everything disappears until refreshed

### DISCUSSION THREAD

**Discussion by DjDaan111** [dani-garcia/vaultwarden#3284] [Q&A]
My trash is quite filled up with a lot of items and I'm going thru 1 by 1, but when I remove an item it will display an empty list until I click to another folder and back to trash or click trash again.

This happens in an up to date Firefox browser with Vaultwarden version 1.27 and webvault Version 2022.12.0, both on my Windows 10 desktop and Windows 11 laptop.

**Comment by BlackDex** [comment]
This probably also happens on vault.bitwarden.com, and is not something we can fix on the server side.

**Comment by zsrv** [comment]
This also happens to me, but in the Bitwarden desktop app (the Linux flatpak version at https://flathub.org/apps/details/com.bitwarden.desktop if it makes any difference).

Version 2023.2.0
Shell 21.3.1
Renderer 106.0.5249.181
Node 16.16.0
Architecture x64

**Comment by BlackDex** [comment]
It tells me it's most likely a client issue.
